NebulaPatcher/Patches/Dynamic/Dedicated_Server_Patch.cs

```diff
@@ -171,15 +171,47 @@ public static bool UICommunicatorIndicatorOnLateUpdate_Prefix()
         return false;
     }
 
-    [HarmonyPostfix]
+    [HarmonyPrefix]
     [HarmonyPatch(typeof(PlanetATField), nameof(PlanetATField.RecalculatePhysicsShape))]
-    public static void RecalculatePhysicsShape_Postfix(PlanetATField __instance)
+    public static bool RecalculatePhysicsShape_Prefix(PlanetATField __instance)
     {
-        // vanilla use GPU to calculate the shape of shield that is not fully covered the whole planet
-        // In this patch it will act as it has been fully covered to make the planet shield effective
-        if (!__instance.isEmpty)
+        // If we're the server, let's update the planet shields manually.
+        // This is required as checks to planetary shields fail, as there is no GPU to simulate the shields
+        // they become `isEmpty = true`.  This causes some checks to fail such as:
+        //  - Relays still landing on a planet when a shield is online
+        //  - Some space to surface weaponry ignoring the shields
+        if (Multiplayer.IsDedicated)
```

Comment:
There is no need for testing Multiplayer.IsDedicated as Dedicated_Server_Patch only patches in dedicated server mode.

Reply:
Oh yes, my apologies - it has been removed.